ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -247,50 +283,50 @@ class Audio(Media):
     in `ingestion_server.es_mapping`.
     """
 
+    class Durations(Enum):
+        """
+        Maximum threshold for each audio duration band
+
+        These durations are also hardcoded in the `duration` field in
+        openverse-api/catalog/api/serializers/audio_serializers.py.
+        """
+        SHORT = 10 * 60 * 1e3  # 10 minutes
```

Comment:
Youtube has the following classification:
- under 4 minutes
- 4-20 minutes
- over 20 minutes.
We might also consider making "Short" mean under 3 minutes, so that it would mainly match sound effects like many of the freesound.org sounds, and "Medium" would mean mostly songs or short talks/podcasts etc.

Reply:
Thanks! I was just shooting in the dark with the time ranges.